### PR TITLE
Add shivank01.md

### PR DESCRIPTION
**Pull request by shivank01**
Please review

### CODE REVIEW

File: _contributors-app/shivank01.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+ - I am Secretary of the Association for Computing Machinery(ACM) IIIT-bh Student Chapter.
+
+ - I am a member of the Programming Society of IIIT-Bh. Together we try to guide the first years in getting started with software development and aim to build the coding culture of our college.
+
+ - I've successfully organized a two-day bootcamp on a basic primer to python and Machine Learning.
+
+ - I am also the Web Administrator of Online Mess Portal 'Rosei' of IIIT Bhubaneswar of which I manage Code, database and server.
+
+ Hackathons and Events: |
```
Comment (palash25) [inline _contributors-app/shivank01.md]: You cannot add your custom fields @shivank01 that is the reason why the tests are failing
Keep this Hackathon section in markdown format in one of the other  fields like volunteer or something

Comment (shivank01) [reply _contributors-app/shivank01.md]: Ok Thanks.

File: _contributors-app/shivank01.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (palash25) [inline _contributors-app/shivank01.md]: You have a merge conflict here

File: _contributors-app/shivank01.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+ - I've successfully organized a two-day bootcamp on a basic primer to python and Machine Learning.
+
+ - I am also the Web Administrator of Online Mess Portal 'Rosei' of IIIT Bhubaneswar of which I manage Code, database and server.
+
+ - Finalist in the team based GDG devfest hackathon. Made a review analysis of products on e-commerce websites using Machine Learning.
+
+ - 4th Runner's up in the pan India Competitive coding contest Kronothon with over 7000 participants.
+---
+||||||| merged common ancestors
```
Comment (palash25) [inline _contributors-app/shivank01.md]: Another conflict marker

Comment (shivank01) [reply _contributors-app/shivank01.md]: ok Will correct it and make it under one commit

**Comment by palash25** [comment]
Pushing again and again with different commits won't help.

Since you have merge conflict markers in your file they will never pass as valid markdown and the travis build will fail

File: _contributors-app/shivank01.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+=======
```
Comment (palash25) [inline _contributors-app/shivank01.md]: another one

**Pull request closed without merging**